saviSimulation/savi/jason_processing/UxV.java

```diff
@@ -63,6 +63,9 @@ public void setPosition(PVector pos) {
 		this.position = pos;
 	}
 	
+	public static void setSeed(Random ran) {
+		rand = ran;
```

Comment:
May want to do a null check here.

Reply:
Should the Random rand be a parameter of WorldObject? Is there a time when you wouldn't need this in a world object?